build/lib/helm_promotion.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+set -o pipefail
+
+export LANG=C.UTF-8
+
+BASE_DIRECTORY=$(git rev-parse --show-toplevel)
+ACCOUNT_ID="${1?Specify first argument - account id}"
+IMAGE_REGISTRY="${2?Specify first argument - image registry}"
```

Comment:
```suggestion
IMAGE_REGISTRY="${2?Specify second argument - image registry}"
```

Reply:
fixed in 17beea798aa9427b7d227eb7d16b9f5b8e48a0e6